src/planscape/conditions/raster_utils.py

```diff
@@ -25,6 +25,42 @@ class ConditionStatistics(TypedDict):
     count: int
 
 
+# A collection of raw raster pixel values.
```

Comment:
what does raw refer to here? it has a special meaning in our condition dataset so maybe we can use a different term?

Reply:
Removed - "raw"'s a redundant.